src/sync/sync.nim

```diff
@@ -100,13 +101,13 @@ proc sync(exercise: Exercise, mode: Mode): Exercise =
 
   result.tests = initExerciseTests(included, excluded, missing)
 
-  writeFile(result)
+  writeTestsToml(result, conf.trackDir)
```

Comment:
Is `conf.trackDir` set to the current directory if the `--track-dir` argument is not specified?

Reply:
It was set to the empty string, which ends up meaning the current directory.

I pushed a new commit, do you prefer it?